web/src/javascript/drawMap.js

```diff
@@ -11,17 +11,16 @@ const screenHeight = screenCanvas.height;
 let xScreenPosOnMap = 0;
 let yScreenPosOnMap = 0;
 
-export const mapWidth = screenWidth * 3;
-export const mapHeight = screenHeight * 3;
+let screenToMapRatio = 0;
```

Comment:
It is set in `initMap`. :)

@Erikad94 is this causing issues *before* `initMap` is called for the first time, however? I'm guessing that a ratio of `0` might cause some errors, for example.